### DISCUSSION TITLE
ZX7D00CE01S 7 inch RGB

### DISCUSSION THREAD

**Discussion by Hooba97** [moononournation/Arduino_GFX#368] [Q&A]
![Screenshot 2023-10-27 062034](https://github.com/moononournation/Arduino_GFX/assets/83105594/db3edcc5-969f-449c-92ae-65679fa19251)
hello, and thank you for adding support for the 7inch screen 

I am doing some prototyping using Arduino IDE on the ESP board 
I can't seem to figure out how to run the screen 

using the PDQgraghic test file, the code runs normally  but the screen stays black 
I'd appreciate the help 

if I am not mistaken I think the problem is from the board selected in the Arduino IDE or its settings 
here's a screenshot

**Comment by moononournation** [comment]
PSRAM should select OSPI

**Reply by Hooba97** [reply]
Do you mean QSPI or OPI 
I tried both , using OPI made it work , thank you 🙏

Follow up question 
I seem to notice some flickering on on black screens. 
Also Why is there a white line at the bottom of the screen ![PXL_20231028_022216737.jpg](https://github.com/moononournation/Arduino_GFX/assets/83105594/c4148a22-cbb4-4c4e-a5c6-ee949951226e)

**Comment by vajpeters** [comment]
Somehow QSPI en OSPI are both not working... Any hints?

**Comment by vajpeters** [comment]
The chip is reporting:

10:24:33.076 -> ESP-ROM:esp32s3-20210327
10:24:33.076 -> Build:Mar 27 2021
10:24:33.076 -> rst:0x1 (POWERON),boot:0x8 (SPI_FAST_FLASH_BOOT)
10:24:33.076 -> SPIWP:0xee
10:24:33.076 -> mode:DIO, clock div:1
10:24:33.076 -> load:0x3fce3808,len:0x44c
10:24:33.076 -> load:0x403c9700,len:0xbd8
10:24:33.076 -> load:0x403cc700,len:0x2a80
10:24:33.076 -> entry 0x403c98d0
10:24:34.171 -> 
10:24:34.171 -> 
10:24:34.171 -> ================================
10:24:34.171 -> Chip Model: ESP32-S3
10:24:34.171 -> Chip version: 0
10:24:34.171 -> Numer of cores: 2
10:24:34.171 -> Flash Chip Size: 16777216
10:24:34.171 -> Flash Chip Speed: 80000000
10:24:34.171 -> ================================

**Comment by vajpeters** [comment]
![settings](https://github.com/moononournation/Arduino_GFX/assets/19168848/2dd59e85-88c4-4ae5-b94a-2c22cd57e556)
Ok, this seems to work on my side...
